### PR TITLE
Fixed environment variables in Cron Update PHP Modules

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Fixed environment variables in Cron Update PHP Modules
| Type?             | bug fix
| Category?         | PM
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | N/A
| How to test?      | N/A

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/26530)

### CODE REVIEW

File: .github/workflows/cron_php_update_modules.yml [hunk 1]
```diff
@@ -21,7 +23,9 @@ jobs:
         run: composer install --prefer-dist
 
       - name: Update PrestaShop packages
-        run: echo "PULL_REQUEST_BODY=$(php .github/workflows/cron_update_modules/script.php)" >> $GITHUB_ENV
```
Comment (PierreRambaud) [inline .github/workflows/cron_php_update_modules.yml:24]: The problem come from that you didn't escape  `\n` 

```
run: echo "PULL_REQUEST_BODY='$(php .github/workflows/cron_update_modules/script.php)'" >> $GITHUB_ENV
```
should have been enough (to try)

File: .github/workflows/cron_update_modules/script.php [hunk 2]
```diff
@@ -126,4 +126,4 @@ function fetchVersionComposerLock(): array {
     . '| How to test?      | N/A' . PHP_EOL
     . '| Possible impacts? | N/A' . PHP_EOL
 ;
-echo $pullRequestBody;
+putenv("SCRIPT_PULL_REQUEST_BODY=$pullRequestBody");
```
Comment (PierreRambaud) [inline .github/workflows/cron_update_modules/script.php]: You will have the same problem as previous :thinking: The text is not encapsulate.

**Review by PierreRambaud — COMMENTED** [review]
Just a little comment

File: .github/workflows/cron_php_update_modules.yml (hunk 1 above)
Comment (Progi1984) [reply .github/workflows/cron_php_update_modules.yml:24]: We can try it.

Comment (Progi1984) [reply .github/workflows/cron_php_update_modules.yml:24]: Thanks

File: .github/workflows/cron_update_modules/script.php [hunk 3]
```diff
@@ -85,7 +85,7 @@ function fetchVersionComposerLock(): array {
     if (!in_array(VENDOR_NAME . $moduleGit['path'], $modulesComposer)) {
         continue;
     }
-    echo PHP_EOL . ' === Module prestashop/' . $moduleGit['path'] . PHP_EOL;
+    // echo PHP_EOL . ' === Module prestashop/' . $moduleGit['path'] . PHP_EOL;
```
Comment (PierreRambaud) [inline .github/workflows/cron_update_modules/script.php]: :thinking:

**Review by sowbiba — APPROVED** [review]
YamlLinter again

**Review by PierreRambaud — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thank you @Progi1984

**Pull request merged**